src/scheduler/Makefile.am

```diff
@@ -121,7 +121,7 @@ common_libs = \
 	@KRB5_LIBS@
 
 pbs_sched_CPPFLAGS = ${common_cppflags}
-pbs_sched_LDADD = ${common_libs}
+pbs_sched_LDADD = ${common_libs} @libundolr_lib@
```

Comment:
I like this change, I suggest doing the same change for server, mom, and comm also.

Reply:
Yeah, I did changes testing it now, will push the commit.

Reply:
I have now pushed the commit